### PR TITLE
Sensor sample app

### PR DESCRIPTION
**Pull request by jlight1961**
New sample app

### CODE REVIEW

File: DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Service/SensorService.cs [hunk 1]
```diff
@@ -34,19 +34,19 @@ public SensorService(SensorRepository repository, SensorViewModel viewModel) : b
...
 
         public async override Task<string> Connect(IDictionary<string, object> message)
         {
             var result = await base.Connect(message);
 
-            await _viewModel.ClearOutput();
-            await _viewModel.SetStatus("NotReady");
-            await _viewModel.SetApiVersion(message.ValueOrDefault("Version", "0.0").ToString());
-            await _viewModel.SetAppVersion(message.ValueOrDefault("AppVersion", "0.0").ToString());
+            _viewModel.ClearOutput();
```
Comment (sharaaU7) [inline DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Service/SensorService.cs]: are these not async anymore?

File: DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Service/SensorService.cs [hunk 2]
```diff
@@ -61,14 +61,33 @@ public override async Task OnEvent(IDictionary<string, object> message)
...
                     await _viewModel.AppendOutputLine("Status: " + status);
                 }
             }
             else
                 await _viewModel.AppendOutputLine(" Event: " + eventName);
         }
 
+        public override Task OnCommand(string command, dynamic parms)
+        {
+            if(command == "Measurement System")
```
Comment (sharaaU7) [inline DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Service/SensorService.cs]: is there a glossary of commands (to understand which all commands are there, and what are capabilities of each of those commands) in the app sample?

File: DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Service/SensorService.cs [hunk 3]
```diff
@@ -61,14 +61,33 @@ public override async Task OnEvent(IDictionary<string, object> message)
...
+            {
+                try
+                { 
+                    var val = parms.MeasurementSystem;
+
+                    this.ViewModel.IsMetric = val.ToString() == "Metric";
```
Comment (sharaaU7) [inline DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Service/SensorService.cs]: should this be case insensitive?

File: DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Repository/SensorChannel.cs [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+using Microsoft.DataStreamer.UWP;
+
+namespace Microsoft.DataStreamer.Samples.SensorSimulator
+{
+    public class SensorChannel : Channel
+    {
+        [JsonIgnore]
```
Comment (sharaaU7) [inline DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Repository/SensorChannel.cs]: too much spacing

File: DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Service/SensorService.cs (hunk 1 above)
Comment (jlight1961) [reply DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Service/SensorService.cs]: Nope

File: DataStreamerUWP/Service/DataStreamerAppService.cs [hunk 5]
```diff
@@ -76,16 +77,23 @@ private async void OnRequestReceived(AppServiceConnection sender, AppServiceRequ
...
+                            // Run asynchronously
+                            _service.Disconnect().FireAndForget();
                             break;
 
                         default:
                         {
-                            returnData.Add("Status", "Fail: unknown command");
+                            var jsonParams = message.ValueOrDefault("Params", "").ToString();
+                            var parms = jsonParams == null ? null : JObject.Parse(jsonParams);
+
```
Comment (sharaaU7) [inline DataStreamerUWP/Service/DataStreamerAppService.cs]: unwanted newline

File: DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Service/SensorService.cs (hunk 3 above)
Comment (jlight1961) [reply DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Service/SensorService.cs]: No need. It's a dropdown on the DS side

File: DataStreamerUWP/Service/DataStreamerAppService.cs (hunk 5 above)
Comment (jlight1961) [reply DataStreamerUWP/Service/DataStreamerAppService.cs]: Don't see what you're talking about but I did remove the commented out line.

File: DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Repository/SensorChannel.cs (hunk 4 above)
Comment (jlight1961) [reply DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Repository/SensorChannel.cs]: Fixes

File: DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Service/SensorService.cs (hunk 2 above)
Comment (jlight1961) [reply DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Service/SensorService.cs]: These commands are defined in the manifest

File: DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Service/SensorService.cs [hunk 6]
```diff
@@ -34,19 +34,19 @@ public SensorService(SensorRepository repository, SensorViewModel viewModel) : b
...
         public CoreDispatcher  Dispatcher { set { _viewModel.Dispatcher = value; }}
 
-        public async Task ClearOutput()
+        public void ClearOutput()
         {
-            await _viewModel.ClearOutput();
+            _viewModel.ClearOutput();
         }
 
         public async override Task<string> Connect(IDictionary<string, object> message)
```
[the remark this answers is not in the document]
Comment (jlight1961) [reply DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Service/SensorService.cs]: No

[the remark this answers is not in the document]
Comment (sharaaU7) [reply DotNet UWP Samples/Sensor Simulator/Sensor Simulator/Service/SensorService.cs]: ping me when you have a new iteration out.

**Pull request merged**